configure_git_repository.md

```diff
@@ -16,29 +16,28 @@ Please stick to it unless you have special needs.
   * Add these rules for the two branches `develop` and `main`:
     * Require pull request reviews before merging
     * Require status checks to pass before merging (after you configured the CI add it to the required checks)
-    * Require branches to be up to date before merging
+
```

Comment:
Sneaky :wink: but IIRC we discussed this as well.

Reply:
Maybe we should mention that one should enable "Always suggest updating pull request branches"?
![image](https://user-images.githubusercontent.com/1435346/203769971-3ee57eae-86ad-452a-9c68-0f02c50ef80f.png)

Reply:
This is a very good idea!